ocsp/responder.go

```diff
@@ -19,6 +20,7 @@ import (
 
 	"github.com/cloudflare/cfssl/log"
 	"github.com/jmhodges/clock"
+	"github.com/cloudflare/cfssl/certdb"
```

Comment:
Alternatively, you can use [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports) to handle import management automatically; it's a drop-in replacement for `gofmt`.